app/src/main/java/org/listenbrainz/android/service/ListenScrobbleService.kt

```diff
@@ -29,17 +30,23 @@ class ListenScrobbleService : NotificationListenerService() {
     @Inject
     lateinit var scrobbleManager: ScrobbleManager
     
-    private var sessionManager: MediaSessionManager? = null
     private var sessionListener: ListenSessionListener? = null
     private var listenServiceComponent: ComponentName? = null
     private val scope = MainScope()
+    
     private val nm by lazy {
         ContextCompat.getSystemService(
             this,
             NotificationManager::class.java
         )!!
     }
-
+    private val sessionManager: MediaSessionManager by lazy {
+        ContextCompat.getSystemService(
+            this,
+            MediaSessionManager::class.java
+        )!!
```

Comment:
This could cause a crash

Reply:
Called using lazy, should be same as calling in-place.

Reply:
```kotlin
private val sessionManager: MediaSessionManager? by lazy {
    val manager = ContextCompat.getSystemService(this, MediaSessionManager::class.java)
    if (manager == null) {
        Log.e("MediaSessionManager is not available in this context.")
    }
    manager
}
```
something like this is good. Avoid `!!` as much as you can